Atomic/backends/_docker.py

```diff
@@ -328,7 +329,8 @@ def pull_image(self, image, remote_image_obj, **kwargs):
         local_image = self.has_image(image)
         if local_image is not None:
             if self.already_has_image(local_image, remote_image_obj):
-                raise ValueError("Latest version of {} already present.".format(image))
+                util.write_err("Latest version of {} already present.".format(image))
```

Comment:
Should we create an exception that handles this, rather then writing the code here.

Something like

raise SuccessException("Latest version of {} already present.".format(image))
￼
That way it could be handled potentially by something like atomic-dbus or other callers.